agents/coder.js

```diff
@@ -5,13 +5,42 @@ function formatCode(files) {
   let code = '';
   files.forEach(file => {
```

Comment:
This function is no longer needed. Was added when we were putting multiple files in the same prompt. You can send file.code and file.path directly to values. Check the PR that I opened recently, where I changed this.

Reply:
I was thinking due to size/token limits this might be a good idea to keep
if we do multi-file changes it's avgFileSize*numOfFiles*2+prompt
Same reason we don't just load all the files in the project into one prompt without summaries.
Keeping this would support more tasks unless we dynamically chunk...

Reply:
changed files->file in 3 places to clean this up